### DISCUSSION TITLE
0.9.1

### DISCUSSION THREAD

**Discussion by sircharlo** [sircharlo/meeting-media-manager#100] [New releases]
Here we go, September's [release](https://github.com/sircharlo/jw-meeting-media-fetcher/releases/tag/v0.9.1), and a bit earlier than planned at that!
There are a few goodies in this one:
- Added a gradual fade-out when background music is stopped
- Added background music auto-stop feature, both time-based and meeting-start-time-based
- Converted MP4s now replace the original media, instead of going into a "Zoom" folder 
- Dramatically increased the quality of converted MP4 files
- Implemented a workaround to account for an oddity in Zoom where, if a converted MP4's resolution was exactly 720p, Zoom was unable to share it
- Fixed an issue where the song list would not get refreshed after changing the language in Settings
- Updated app strings, translations and docs
- Minor visual tweaks, bugfixes and various code improvements

**Comment by s0q** [comment]
Love these features, thank you!

**Comment by mjacobus** [comment]
Good stuff!
